HTTPC-146: Document connect timeout when a host name resolves multiple IPs (#1604)
mulesoft/docs-connectors: http/1.5/modules/ROOT/pages/http-documentation.adoc

[%header,cols="20s,25a,30a,15a,10a"]
|===
| Field | Type | Description | Default Value | Required

> | Connection Timeout a| Number | Time to wait during a connection to a remote server before failing with a timeout. If the requester host configured resolves to multiple IPs, the host tries to connect to every IP.  Therefore, if every connection times out, the number of resolved IPs multiplies the global connection timeout. | 30000 |

Reviewer: ```suggestion
| Connection Timeout a| Number | Time to wait during a connection to a remote server before failing with a timeout. If the configured requester host resolves to multiple IP addresses, the host tries to connect to every single IP address.  Therefore, if every connection times out, the number of resolved IP addresses multiplies the global connection timeout. | 30000 |
```

> | Send Tcp No Delay a| Boolean | Indicates whether or not to collect the transmitted data for greater efficiency and sent the data immediately. | true |

Reviewer: ```suggestion
| Send Tcp No Delay a| Boolean | Indicates whether or not to collect the transmitted data for greater efficiency and send the data immediately. | true |
```

| Linger a| Number | How long, in milliseconds, the socket takes to close, so any remaining data is transmitted correctly. |  |
| Keep Alive a| Boolean | Indicates whether or not to close the open socket connection. | false |
| Fail On Unresolved Host a| Boolean | Indicates whether the socket fails during its creation if the host set on the endpoint cannot be resolved. | true |

> | Send Buffer Size a| Number | Size of the buffer in bytes when sending data. |  |

Reviewer: ```suggestion
| Send Buffer Size a| Number | Size of the buffer, in bytes, when sending data. |  |
```

> | Receive Buffer Size a| Number | Size of the buffer in bytes when receiving data. |  |

Reviewer: ```suggestion
| Receive Buffer Size a| Number | Size of the buffer, in bytes, when receiving data. |  |
```
